src/test/java/com/google/api/codegen/testdata/discoveries/storagetransfer.v1.json.overrides

```diff
@@ -13,6 +13,7 @@
               "isArray": false,
               "isMessage": false
             },
+            "placeholder": "",
```

Comment:
So this is because we would generate the wrong placeholder otherwise, until #494 is fixed?